app/src/main/res/layout/list_item_storage_root.xml

```diff
@@ -41,7 +41,7 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginTop="8dp"
-        android:layout_marginEnd="16dp"
+        android:layout_marginEnd="40dp"
```

Comment:
I think this is the answer why you set the end margin of the recyclerview to 0, because the padding it is the items. To be honest, I think it is strange that this is mixed like this. The end padding comes from the items, but the start from the list? Is this an SUW thing?

Reply:
same as above, c)